Source/Fuse.Controls.Video/Video.ScriptClass.uno

```diff
@@ -0,0 +1,76 @@
+using Uno;
+using Uno.UX;
+
+using Fuse.Scripting;
+
+namespace Fuse.Controls
+{
+	public partial class Video
+	{
+		static Video()
+		{
+			ScriptClass.Register(typeof(Video),
+				new ScriptMethod<Video>("getPosition", getPosition, ExecutionThread.Any),
+				new ScriptMethod<Video>("getDuration", getDuration, ExecutionThread.Any),
+				new ScriptMethod<Video>("resume", resume, ExecutionThread.MainThread),
+				new ScriptMethod<Video>("pause", pause, ExecutionThread.MainThread),
+				new ScriptMethod<Video>("stop", stop, ExecutionThread.MainThread));
+		}
+
+		/**
+			Gets the current position in the Video in seconds.
+
+			@scriptmethod getPosition()
+		*/
+		static object getPosition(Context c, Video v, object[] args)
+		{
+			var ret = 0.0;
+			lock (v)
+				ret = v._outPosition;
+			return (object)ret;
+		}
+
+		/**
+			Gets the duration of the Video in seconds. getDuration will return 0 until the video has initialized.
+
+			@scriptmethod getDuration()
+		*/
+		static object getDuration(Context c, Video v, object[] args)
+		{
+			var ret = 0.0;
+			lock (v)
+				ret = v._outDuration;
+			return (object)ret;
+		}
+
+		/**
+			Resumes playback from the current position.
+
+			@scriptmethod resume()
+		*/
+		static void resume(Context c, Video v, object[] args)
+		{
```

Comment:
Please put checks on the `args` here (and other functions). If it's not 0-length emit a Diagnostics.UserError.